lib/constants.dart

```diff
@@ -0,0 +1 @@
+export 'src/constants.dart';
```

Comment:
- Why are constants been exposed as public?
- If there is a usecase for exposing them, do all of them need to be exposed considering most if not all are used internally in the package?

Reply:
Since I'll only be using 2 variables from the file I will just declare them in consumer, is that okay?